content/en/docs/concepts/security/api-server-bypass-risks.md

```diff
@@ -0,0 +1,149 @@
+to the data stored inside etcd.
+
+Direct access to etcd is not subject to Kubernetes admission control and is not logged
+by Kubernetes audit logging. An attacker who has read access to the API server's
+etcd client certificate (or can create a new trusted client certificate) can gain
+cluster admin rights by accessing cluster secrets or modifying access rules. Even without
+elevating their privileges, an attacker who can modify etcd can retrieve any API object
+or create new workloads inside the cluster.
+
+Many Kubernetes providers configure 
+etcd to use mutual TLS (both client and server verify each other's certificate for authentication).
+There is no widely accepted implementation of authorization for the etcd API, although
+the feature exists. Since there is no authorization model, any certificate
+with client access can be used to gain full access to etcd. Typically, client certificates
+that are only used for health checking can also grant full read and write access.
+
+One way in which an attacker can make use of stolen etcd credentials is to create a 
+special-purpose API server to manipulate resources in a cluster. All audit events would
+stay with the API server that handled the request and would therefore not be present
+in the audit log for the target cluster. This particular attack would be very difficult
+to detect, and easier to execute than creating data in etcd directly.
+
+### Mitigations {#etcd-api-mitigations}
+
+- Ensure that the certificate authority trusted by etcd is used only for the purposes of
+  authentication to that service.
+- Control access to the private key for the etcd server certificate, and to the API server's
+  client certificate and key.
+- Consider restricting access to the the etcd port at a network level, to only allow access
+  from specified and trusted IP address ranges.
+
+
+## Container runtime socket {#runtime-socket}
+
+On each node in a Kubernetes cluster, access to interact with containers is controlled 
+by the container runtime (or runtimes, if you have configured more than one). Typically,
+the container runtime exposes a Unix socket that the kubelet can access. An attacker with
+access to this socket can launch new containers or interact with running containers.
+
+At the cluster level, the impact of this access depends on whether the containers that
+run on the compromised node have access to Secrets or other confidential
+data that an attacker could use to escalate privileges to other worker nodes or to
+control plane components.
+
+### Mitigations {#runtime-socket-mitigations}
+
+- Ensure that you tightly control access to container runtime sockets. 
+   When possible, restrict this access to the `root` user.
+- Isolate the kubelet from other components running on the node, using
+   mechanisms such as Linux kernel namespaces.
+- Ensure that you restrict or forbid the use of [`hostPath` mounts](/docs/concepts/storage/volumes/#hostpath)
+   that include the container runtime socket.
```

Comment:
also disallow hostPath mounts of parent directories of the socket?

also, can't symlinks on the host filesystem be used to make the kubelet mount arbitrary host paths unless all hostPath volumes are readOnly (https://kubernetes.io/docs/concepts/storage/volumes/#hostpath)?

Reply:
cool, expanded a bit on this